kuma/wiki/urls.py

```diff
@@ -29,6 +29,9 @@
     url(r'^\$files$',
         edit_attachment,
         name='attachments.edit_attachment'),
+    url(r'^\$edit/(?P<revision_id>\d+)$',
+        views.edit.edit,
+        name='wiki.new_revision_based_on'),
```

Comment:
I'm not sure how this `wiki.new_revision_based_on` url made its way back into our lives, but it needs to go! 😄

Reply:
Nice catch!